app/jobs/bulkrax/delete_file_set_job.rb

```diff
@@ -2,4 +2,22 @@
 
 module Bulkrax
   class DeleteFileSetJob < DeleteJob; end
+
+  def perform(entry, importer_run)
```

Comment:
Do you need to pass the importer_run? It doesn't seem like it's getting used here.

Reply:
its used in the parent job (so the call to super gets it)

Reply:
this was a really good catch Shana. its legal to define the method on the module so its syntaxitcally correct but would not have ever been called the way I had it